packages/sitecore-jss/src/i18n/dictionary-service.ts

```diff
@@ -0,0 +1,63 @@
+import mcache from 'memory-cache';
+
+const defaults = {
```

Comment:
I added Object.freeze().
All-caps is really bad for readability and I personally really dislike this convention. It used to be popular before ES2015, when `const` and `let` were not part of the language. Now, and especially when TypeScript is used, there are better ways to show that something is readonly/enum/static/shouldn't be changed.

Reply:
@anastasiya29 Got it, but which convention we are using in project? Because sometimes we are using uppercase letters, and somebody prefer to use uppercase convention.

Reply:
Per our team discussion, this has been renamed to 
```
const DEFAULTS = Object.freeze({
  pageSize: 10,
});
```